src/packet/userid.js

```diff
@@ -74,14 +76,22 @@ class UserIDPacket {
   }
 
   /**
-   * Set userid string from object, e.g. { name:'Phil Zimmermann', email:'phil@openpgp.org' }
+   * Set userId from object
+   * @param {Object} userId  object specifying userId name, email and comment
    */
-  format(userid) {
-    if (util.isString(userid)) {
-      userid = util.parseUserId(userid);
+  fromObject(userId) {
```

Comment:
I meant like this:
```suggestion
  static fromObject(userId) {
```
and then construct a `UserIDPacket` and assign to it inside the function, and call `UserIDPacket.fromObject(userId)` elsewhere (similar to `Message.fromText`)